app/models/manageiq/providers/openstack/manager_mixin.rb

```diff
@@ -12,6 +12,7 @@ module ManageIQ::Providers::Openstack::ManagerMixin
   #
   module ClassMethods
     def raw_connect(password, params, service = "Compute")
+      # TODO: this method seems to be used only for Provider credentials validation
```

Comment:
isn't this used also to with_provider connection, etc? So for all actions?

Reply:
There is a method with the same name in OpenStack Handle, which is used for accessing OpenStack. I didn't find other usage, but it would be great have more eyes checking it!

Reply:
I would expect raw_connect to return the object...and with_provider_connection definitely uses the object returned by connect (which should be obtained from raw_connect.  I don't think returning true here is the right approach.